09-intro-to-bcdata.Rmd

```diff
@@ -17,9 +17,133 @@ knitr_fig_path("09-")
 ```
 
 
-# Introduction to bcdata package
+# Introduction to the bcdata package
+
+The bcdata R package contains functions for searching & retrieving data directly from the B.C. Data Catalogue.
 
 ```{r}
 library(bcdata)
 ```
 
+Let's look in the B.C. Data Catalogue for some data on libraries in British Columbia:
+
+```{r}
+bcdc_search("public libraries")
+```
+
+There are many catalogue records with data relating to libraries. Let's examine the first returned record more closely:
+
+
+```{r}
+bcdc_get_record("bc-public-libraries-systems-branches-and-locations")
+```
+
+Next, we can use the `bcdc_get_data` function to load this data directly into R:
```

Comment:
Do you want to add a note about multiple resources?

Reply:
Yes, I will implement @boshek's suggestion below. I want to demo the prompt for the resource selection when using it interactively.